### PR TITLE
Changes made for migration to python 3

### PR DESCRIPTION
**Pull request by aanil**
Changes made for migration to python3. Also updated the requirements

### CODE REVIEW

File: run_dir/static/js/projects.js [hunk 1]
```diff
@@ -631,10 +631,10 @@ function updateStatusBar1(source, type){
       dealWithDatepickers('datepick3', 'add');
       dealWithDatepickers('datepick2', 'add');
     }
-    if(jQuery.inArray('statusOptOngoing', checklist) !== -1 || jQuery.inArray('statusOptOpen', checklist) !== -1 || jQuery.inArray('statusOptNeedsReview', checklist) !== -1 || jQuery.inArray('statusOptRecCtrl', checklist) !== -1 ){
```
Comment (ewels) [inline run_dir/static/js/projects.js]: This seems like a kind of random change - is it intentional?

Comment (aanil) [reply run_dir/static/js/projects.js]: It was a minor bug which I noticed while testing.

**Comment by aanil** [comment]
> Looking good! Does it still work with Python 2 as well?

I updated tornado to the latest version(6.0.2) which requires Python 3.5.2 or above. We can downgrade to Tornado 5.1 if it has to run on Python 2.7 as well.

**Comment by ewels** [comment]
No, I think this is fine. Just need to be careful with the deployment. Have you tested this on stage?

**Comment by aanil** [comment]
Its running on stage right now! :smile:

**Comment by ewels** [comment]
Nice! If it's all working well then I say go for it - merge and deploy..

TO-INFINITY-AND

**Comment by alneberg** [comment]
Ok, I've went through the code base with `2to3` without finding anything else that needs to be fixed. Let's go for this, but send out an email to the platform that they should keep an extra eye open for any kind of strange behaviour.

**Pull request merged**